komga-webui/src/locales/en.json

```diff
@@ -4,8 +4,8 @@
       "pageText": "{0}-{1} of {2}"
     },
     "dataIterator": {
-      "loadingText": "Loading items...",
```

Comment:
Triple-dot is not an ellipsis. Typographically correct, and it doesn't read "dot-dot-dot" on screen-readers.

Reply:
True. This string is coming from the Vuetify repo directly, but the ellipsis is better.